app/assets/stylesheets/styles/components/guides/_new_guides.css.scss

```diff
@@ -318,11 +323,29 @@
 
   /* Step 2 */
   .life-stages {
+    .all-stages-wrapper {
+      margin-left: auto;
+      margin-right: auto;
+      @media #{$small-only} {
+        width: 360px;
+      }
+      @media #{$medium-only} {
+        width: 540px;
+      }
+      @media #{$large-only} {
+        width: 720px;
+      }
+      @media #{$xlarge-up} {
+        width: 900px;
+      }
+    }
+
     .stage-wrapper {
       display: inline-block;
       overflow: hidden;
-      height: 142px;
       margin-bottom: 3%;
+      height: 150px;
+      width: 180px;
```

Comment:
We probably shouldn't be hard-coding heights & widths like this. Is there a reason for this?